_data/projects/TimelyDiff.yml

```diff
@@ -0,0 +1,13 @@
+name: TimelyDiff
+desc: A JavaScript package that converts Unix timestamps into human-readable time differences.",
+site: https://github.com/artisticLogicMK/timelydiff
+tags:
+  - javascript
+  - js
```

Comment:
@artisticLogicMK - Rename tag 'js' to be 'javascript'
Please remove js tag as per https://github.com/up-for-grabs/up-for-grabs.net/actions/runs/4303614573/jobs/7837282422?pr=3801